File: src/main/java/catwalks/register/MultipartRegister.java
package catwalks.register;

import catwalks.part.PartCatwalk;
import catwalks.part.PartScaffold;
import catwalks.part.PartStair;
import catwalks.part.converter.PartConverterScaffold;
import mcmultipart.multipart.MultipartRegistry;

/**
 * Created by TheCodeWarrior
 */
public class MultipartRegister {
	
	public static void register() {
		// scaffold
		MultipartRegistry.registerPart(PartScaffold.class, PartScaffold.ID);
		PartConverterScaffold converter = new PartConverterScaffold();
		MultipartRegistry.registerPartConverter(converter);
		MultipartRegistry.registerReversePartConverter(converter);
		
		// catwalk
		MultipartRegistry.registerPart(PartCatwalk.class, PartCatwalk.ID);
		
		
		// catwalk
		MultipartRegistry.registerPart(PartStair.class, PartStair.ID);
	}
	
}

File: src/main/java/catwalks/register/MultipartRegister.kt
package catwalks.register

import catwalks.part.PartCatwalk
import catwalks.part.PartScaffold
import catwalks.part.PartStair
import catwalks.part.converter.PartConverterScaffold
import mcmultipart.multipart.MultipartRegistry

/**
 * Created by TheCodeWarrior
 */
object MultipartRegister {

    fun register() {
        // scaffold
        MultipartRegistry.registerPart(PartScaffold::class.java, PartScaffold.ID)
        val converter = PartConverterScaffold()
        MultipartRegistry.registerPartConverter(converter)
        MultipartRegistry.registerReversePartConverter(converter)

        // catwalk
        MultipartRegistry.registerPart(PartCatwalk::class.java, PartCatwalk.ID)

        // stair
        MultipartRegistry.registerPart(PartStair::class.java, PartStair.ID)
    }

}
